R/allele_qc.R

```diff
@@ -1,38 +1,86 @@
-#' Match alleles between summary statistics and SNP information.
-#' 
+#' Converted  Variant ID into a properly structured data frame
+#' @param variant_id A data frame or character vector representing variant IDs.
+#'   Expected formats are a data frame with columns "chrom", "pos", "A1", "A2",
+#'   or a character vector in "chr:pos:A1:A2" or "chr:pos_A1_A2" format.
+#' @return A data frame with columns "chrom", "pos", "A1", "A2", where 'chrom' 
+#'   and 'pos' are integers, and 'A1' and 'A2' are allele identifiers.
+#' @import stats
+#' @export
```

Comment:
Please don't export it by replacing `@export` with `@noRd`. this is an internal function